### PR TITLE
Add OpenMW 0.39.0

### PR DESCRIPTION
**Pull request by benwebber**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download openmw` is error-free.
- [x] `brew cask style --fix openmw` left no offenses.
- [x] `brew cask install openmw` worked successfully.
- [x] `brew cask uninstall openmw` worked successfully.

---

From the [FAQ](https://openmw.org/faq/):

> OpenMW is a free, open source and modern engine based which reimplements and extends the one that runs the 2002 open-world RPG Morrowind.
> […]
> **NOTE:** Playing Morrowind with this engine still requires the Morrowind data files that you own.

### CODE REVIEW

File: Casks/openmw.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+cask 'openmw' do
+  version '0.39.0'
+  sha256 '378d959f9e14b1f943bdd9edc45f1013904ec21632e73d98a745bcbbce0028ba'
+
+  url 'https://github.com/OpenMW/openmw/releases/download/openmw-0.39.0/OpenMW-0.39.0.dmg'
```
Comment (fanquake) [inline Casks/openmw.rb]: You should be using #{version} in the url

Comment (vitorgalvao) [inline Casks/openmw.rb]: `url "https://github.com/OpenMW/openmw/releases/download/openmw-#{version}/OpenMW-#{version}.dmg"`

Comment (vitorgalvao) [inline Casks/openmw.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

Comment (vitorgalvao) [inline Casks/openmw.rb]: Needs a `# github.com/OpenMW/openmw was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

File: Casks/openmw.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  name 'OpenMW'
+  homepage 'https://openmw.org/'
+  license :gpl
+
+  suite 'OpenMW'
+
+  caveats <<-EOS.undent
```
Comment (vitorgalvao) [inline Casks/openmw.rb]: Please remove this. It is not related to the installation process itself.

**Comment by benwebber** [comment]
Thank you for the feedback. It's been a while since I've added a cask.

I've made the requested changes and squashed my commits.

**Comment by vitorgalvao** [comment]
Looks good. Thank you.

**Pull request merged**